src/pg/sql/40_observatory_utility.sql

```diff
@@ -0,0 +1,237 @@
+
+-- Returns a list of avaliable geometry columns
+CREATE OR REPLACE FUNCTION OBS_ListGeomColumns()
+  RETURNS TABLE(column_id text)
+AS $$
+  SELECT id FROM observatory.OBS_column WHERE type ILIKE 'geometry';
+$$ LANGUAGE SQL IMMUTABLE;
+
+-- Returns the table name with geoms for the given geometry_id
+-- TODO probably needs to take in the column_id array to get the relevant
+-- table where there is multiple sources for a column from multiple
+-- geometries.
+CREATE OR REPLACE FUNCTION OBS_GeomTable(
+  geom geometry,
+  geometry_id text
+)
+  RETURNS TEXT
+AS $$
+DECLARE
+  result text;
+BEGIN
+  EXECUTE '
+    SELECT tablename FROM observatory.OBS_table
+    WHERE id IN (
+      SELECT table_id
+      FROM observatory.OBS_table tab,
+           observatory.OBS_column_table coltable,
+           observatory.OBS_column col
+      WHERE type ILIKE ''geometry''
+        AND coltable.column_id = col.id
+        AND coltable.table_id = tab.id
+        AND col.id = $1
+        AND ST_Intersects($2, ST_SetSRID(bounds::box2d::geometry, 4326))
```

Comment:
not sure, but wouldn't `$2 && bounds` just work? what's the type and SRID of `bounds`?